### PR TITLE
Cleanup config and tests

### PR DESCRIPTION
**Pull request by bcavagnolo**

### CODE REVIEW

File: datalake/conf.py [hunk 1]
```diff
@@ -7,39 +10,69 @@
...
+        option = get_click_option_args(self.name)
+        @click.option(*option['args'], **option['kwargs'])
+        def wrapped_f(*args, **kwargs):
+            f(*args, **kwargs)
+        return wrapped_f
+
+
+config_parser = ArgParser(default_config_files=['/etc/datalake.conf'])
+
+for k, v in config_schema.items():
```
Comment (isilien) [inline datalake/conf.py]: Since keys don't need to be copied, `config_schema.values()` instead of `.items()`?

File: test/__init__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        'version': 0,
+        'start': 1426809600000,
+        'end': 1426895999999,
+        'where': 'nebraska',
+        'what': 'apache',
+        'hash': '12345'
+    }
+
+def random_word(length):
+    return ''.join(random.choice(string.lowercase) for i in range(length))
```
Comment (isilien) [inline test/__init__.py]: `xrange` instead of `range`?

File: test/__init__.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        'where': random_word(10),
+        'what': random_word(10),
+    }
+
+@pytest.fixture
+def tmpfile(tmpdir):
+    name = random_word(10)
+    def get_tmpfile(content):
+        f = tmpdir.join(name)
+        f.write(content)
```
Comment (isilien) [inline test/__init__.py]: `f.close()`?

Comment (bcavagnolo) [inline test/__init__.py]: Good catch. But this is actually a py.test "LocalPath" and it doesn't have a .close method. I was also surprised.

**Pull request merged**